DaveeHorvath/ascension
# Initial draft for Madfcat/backend

Added 1 endpoint for API.
User creation. Player model is extending built-in User model in Django.
Also, when the Player record is removed, a User is removed as well.
The Player has display name and will have avatar in the future.

The idea is to use the built-in Django authentication. That's why the Player model is built on top of the User model.

To test:

```
docker compose up
```

Go to:
http://localhost:8000/admin/

login: admin
password: adminpassword

Credentials can be changed from `.env`

To create a user check the attached screenshot.
<img width="490" alt="image" src="https://github.com/user-attachments/assets/0fd12ff9-16ac-49e8-8bc4-f379282092f4">
